tests/test_make_job_config.py

```diff
@@ -37,3 +37,14 @@ def test_container_name_can_be_set_by_caller(self):
         job_config = make_job_config(load_job_spec(), container_name='mycoolcontainer')
         container_name = job_config['spec']['template']['spec']['containers'][0]['name']
         self.assertEqual('mycoolcontainer', container_name)
+
+    def test_resources_being_passed(self):
+        test_resources = {
+                "requests": {
+                    "cpu": 1,
```

Comment:
I think I should delete this CPU request as we know there are errors with doing CPU resource limits and it may give end users a false sense of security